src/main/java/vazkii/quark/tweaks/feature/DispensersDyeBlocks.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * This class was created by <ichttt>. It's distributed as
+ * part of the Quark Mod. Get the Source Code in github:
+ * https://github.com/Vazkii/Quark
+ *
+ * Quark is Open Source and distributed under the
+ * CC-BY-NC-SA 3.0 License: https://creativecommons.org/licenses/by-nc-sa/3.0/deed.en_GB
+ *
+ * File Created @ [16/03/2017, 16:32:53 (CET)]
+ */
+package vazkii.quark.tweaks.feature;
+
+import net.minecraft.block.*;
+import net.minecraft.dispenser.IBlockSource;
+import net.minecraft.init.Bootstrap;
+import net.minecraft.init.Items;
+import net.minecraft.item.EnumDyeColor;
+import net.minecraft.item.ItemDye;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.tileentity.TileEntityShulkerBox;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import vazkii.quark.base.module.Feature;
+
+import javax.annotation.Nonnull;
+
+public class DispensersDyeBlocks extends Feature {
+    @Override
+    public void init(FMLInitializationEvent event) {
+        BlockDispenser.DISPENSE_BEHAVIOR_REGISTRY.putObject(Items.DYE, new DyeBehaviour());
+    }
+
```

Comment:
It's already there at the buttom of the class, but my formatting is really terrible i'll fix that